test/test_utils.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import subprocess
+import tempfile
+
+
+def run_and_get_output(command):
+    tmp = tempfile.NamedTemporaryFile()
```

Comment:
`with tempfile.NamedTemporaryFile() as tmp`: or something like that and remove the call to `close`